src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -219,16 +218,21 @@ export default class SummaryBarGraph extends React.Component<ISummaryBarGraphPro
                                 }
                                 return obj;
                             }, {} as any)
-                            return _.reduce(alterationCounts, (arr, value:{count:number, backgroundColor: string}, label) => {
+                            const reducedAlterations = _.reduce(alterationCounts, (arr, value:{count:number, backgroundColor: string}, label) => {
                                 if (value.count) {
                                     arr.push({
                                         text: this.getLegendNames(label),
-                                        fillStyle: value.backgroundColor,
-                                        index: counter ++
+                                        fillStyle: value.backgroundColor
                                     });
                                 }
                                 return arr;
                             }, [] as any);
+                            const orderedAlts = ['Multiple Alterations', 'Amplification', 'Mutation', 'Fusion', 'Gain',
```

Comment:
it would be best to declare this somewhere else so that it's not rebuilt on every function call